src/emc/usr_intf/gmoccapy/gmoccapy.py

```diff
@@ -146,12 +146,12 @@ def excepthook(exc_type, exc_obj, exc_tb):
 def _find_valid_icon_themes():
     valid_icon_themes = [
         # path, name
-        (None, "none")
```

Comment:
Removing the "None" option leads to a probably unwanted problem: When having any icon theme (even no linux-cnc compatible - like a generic desktop icon theme) in the `USER_ICON_THEME_DIR` ("~/.icons") and no specific preference set, the first icon theme from `USER_ICON_THEME_DIR` gets picked up and most likely is incompatible (missing icons).

I can think of two ways to address this:
1. Reverse the order of directories to first load from the ICON_THEME_DIR
2. Assign a fixed default theme (classic for example)

Reply:
I would go for option 2